automation/test.sh

```diff
@@ -37,6 +37,13 @@ export NAMESPACE="${NAMESPACE:-kube-system}"
 trap '{ make cluster-down; }' EXIT
 
 make cluster-down
+
+# Set environment variables if windows image directory exists
+if [ -d "/nfs/data/windows2016" ]; then
```

Comment:
Can we make  NFS_WINDOWS_DIR configurable? Something like `${NFS_WINDOWS_DIR:-/nfs/data/windows2016}`

Reply:
fixed